outlines/models/transformers.py

```diff
@@ -100,6 +116,24 @@ def convert_token_to_string(self, token: str) -> str:
 
         return string
 
+    def get_messages(self, prompt: str) -> List[Dict[str, str]]:
+        """
+        Template for one-turn chat.
+        """
+        return (
+            [self.SYSTEM_MESSAGE.copy(), self.USER_MESSAGE_TEMPLATE(prompt)]
+            if self.SYSTEM_MESSAGE is not None
+            else [self.USER_MESSAGE_TEMPLATE(prompt)]
+        )
+
+    def apply_chat_template(self, prompt: str) -> str:
+        messages = self.get_messages(prompt)
```

Comment:
We should probably have this be part of `class Tokenizer` and raise `NotImplementedError` by default.